src/colorizer/utils/url_utils.ts

```diff
@@ -77,13 +77,14 @@ export function fetchWithTimeout(
  * @param track integer track number.
  * @param time integer frame number. Ignores values where `time <= 0`.
  */
-export function saveParamsToUrl(
+export function getUrlParams(
```

Comment:
Separated this out into two functions for unit testing!

Reply:
The deal you made here is that you gotta write a test now. 😁

Reply:
How could I do this to me 😭

Reply:
Test have been written! It doesn't cover everything in url_utils.ts because I expect to be overhauling it soon when I refactor collections. (famous last words)